spring-web/src/main/java/org/springframework/web/jsf/el/SpringBeanFacesELResolver.java

```diff
@@ -122,9 +122,7 @@ public boolean isReadOnly(ELContext elContext, @Nullable Object base, Object pro
 		if (base == null) {
 			String beanName = property.toString();
 			WebApplicationContext wac = getWebApplicationContext(elContext);
-			if (wac.containsBean(beanName)) {
-				return true;
-			}
+			return wac.containsBean(beanName);
```

Comment:
Although the proposed changed is technically correct, please revert this for the sake of readability.